packages/eslint-plugin/lib/__tests__/rename-prop.test.ts

```diff
@@ -0,0 +1,164 @@
+import { RuleTester } from 'eslint';
+
+import rule, { UpdatePropNameOptions } from '../rules/rename-prop';
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    sourceType: 'module',
+    ecmaVersion: 'latest',
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+});
+
+ruleTester.run('jsx/update-prop-name', rule, {
```

Comment:
How do you see `jsx/update-prop-name` being used? Is this something someone would configure in their eslintrc, run and then remove from the config? 

Or maybe would they have multiple configurations setup, one for modal-dialog, one for button etc then they remain in the eslintfile and can be updated as the version changes?

Reply:
I think yeah it could run alongside / with / in place of / any codemod approach. Maybe a codemod could even be used to update the eslint config around releases. The flexibility of the current config is that it can be used with lots of configs and thus different modules.

The difficult part to figure out is when to turn it off or how to manage ratcheting it.